tests/seq/tsequtils.nim

```diff
@@ -1,64 +0,0 @@
-discard """
```

Comment:
Where have the tests from these been moved to?

Reply:
[tsequtils block inside of tseq.nim](https://github.com/nim-lang/Nim/pull/9368/files/7517b36058520d668c28e197df4f0e104ace6051#diff-47f3359be6315fc4967ba009c106afcaR110)